### ISSUE TITLE
CivicPy cache wrapper

### ISSUE THREAD

**Issue by jsstevenson** [cancervariants/metakb#303]
Idle thought while reviewing #302 -- given that our data pipeline is reliant on specific checkpoints of CivicPy data, it might be helpful to write a small function or two that fetches the cache pickle for a relevant date (eg when an analysis notebook or a test set was run) from a VICC S3 bucket and places it in the appropriate local file location. That way we can easily ensure that team members are working with the same data set, analyses are reproducible, etc.

**Comment by korikuzma** [comment]
@jsstevenson The civicpy cache can be found in s3://vicc-metakb/civicpy/

**Comment by jsstevenson** [comment]
Right, I'm thinking about a way to declare/ensure in code that you're using a specific date's checkpoint file. We could also just use bash calls in a jupyter notebook too of course.

**Comment by korikuzma** [comment]
@jsstevenson Yes, I like that! Just thought I'd post the location for whoever grabs the issue (not sure if everyone on the MetaKB team knew we also stored the civicpy cache in this bucket)

**Comment by jsstevenson** [comment]
@korikuzma sounds like a job for some documentation!

**Comment by korikuzma** [comment]
@jsstevenson https://github.com/cancervariants/metakb/issues/198#issuecomment-2020211903
